### PR TITLE
completes week 2 assessment

### PR DESCRIPTION
**Pull request by fmarte23**
# Submitting Week 2 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [x] Career Readiness
- [x] Coding Challenges
- [x] Coding Critique
- [x] Coding Concepts
- [x] Interview Questions

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here: Having trouble with jest. I need to ask help running this because when I tried it, it ran other files and not my specific codes.

### CODE REVIEW

File: career-readiness.md [hunk 1]
```diff
@@ -8,17 +8,17 @@ This week, we covered the importance of a great technical portfolio to help you
 
 - Spend time looking at other developer’s portfolios. Submit the links to three portfolios you will use for inspiration.
 
-1.
-2.
-3.
+1. https://www.linkedin.com/in/chris-aument/
+2. https://www.linkedin.com/in/ashley-lappies/
+3. https://www.linkedin.com/in/jacob-ross96/
```
Comment (chrisyhlee) [inline career-readiness.md:13]: This was asking for links to portfolios, not linkedin

File: code-challenges.test.js [hunk 2]
```diff
@@ -13,39 +13,102 @@
...
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
+// // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+
+// NOTE: I am having trouble running my codes on jest. I followed the instructions from the syllabus, but it keeps running test that is not specific to this week's assessment. I have the codes below, but I will need to seek help from one of the instructors regarding this.
```
Comment (chrisyhlee) [inline code-challenges.test.js:24]: You have code written, but you don't have any tests created. Remember, jest tests use `describe`, `it`, and `expect` statements. Also, I would make sure you ran `yarn add jest`. If you didn't run that command in the terminal, you won't be able to run jest.

**Review by chrisyhlee — APPROVED** [review]
This assessment displays Novice. I suggest looking over jest as well as paying attention to what the prompt is asking. For example, on code challenge 2, it asked you to create a function, but you created a variable. Like I wrote in the comments, to troubleshoot why jest isn't working, it could be because you didn't run yarn add jest in the terminal. If you did, then it's because you didn't actually write any tests. Make sure to revisit jest testing and pay attention during assessment review!

**Comment by fmarte23** [comment]
Thank you Chris. I was able to attend this week's assessment review and learned a lot. Also, I have figured out my jest problem after driving a lot in this week's challenges.

**Pull request merged**